src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -390,17 +390,18 @@ private Task BuildTreeForSnapshotAsync(IDependenciesSnapshot snapshot)
             }
 
             var nowait = SubmitTreeUpdateAsync(
```

Comment:
@lifengl I noticed this existing "nowait" pattern here - should we be Forking/FireAndForgeting this thing?